### ISSUE TITLE
Not working on SDK 6.0.0.GA

### ISSUE THREAD

**Issue by cayaumas** [Sitata/titanium-google-analytics#61]

**Comment by cayaumas** [comment]
I am new building modules on Appcelerator, to update the module i do the following:

- Change project file structure, generating new module project and add the module code
- NDK r12b (suggested by Appcelerator support)
- Update manifest (like this https://github.com/Sitata/titanium-google-analytics/pull/59)

Re package module successfully, but when is included on app the app fails to build with this error:
fail to sign apk duplicate entry: assets/README

Titanium SDK 6.0.0.GA
Mac OS X El Capitan 10.11.6
Testing on Android 6.0.1

Let me know if you want other information

Thanks

**Comment by jovevskipetar** [comment]
@cayaumas Just remove assets/README and rebuild the module. If you have the problem again try to update mobilesdk/osx/6.0.0.GA/android/cli/commands/_buildModule.js with this commit https://github.com/appcelerator/titanium_mobile/pull/8551/files

AndroidNDK: android-ndk-r11c
Titanium SDK: 6.0.0.GA

**Comment by cayaumas** [comment]
Thanks for your answer @jovevskipetar, the module builds but not work when is requiere the app crash.

@jovevskipetar you try to rebuild this module? and you recommend use ndk r11c?

**Comment by jovevskipetar** [comment]
@cayaumas Yes the module builds without problems with old _buildModule.js gen folder contains some files that makes problems in the app where we are installing the module. New _buildModules.js removes that files. Next step you need to do is to remove assets/README file. And app will be start without problems. 

I spend a week to investigate this problem and rebuild this module and another 3 modules. For me r11c works fine.

**Comment by clemblanco** [comment]
Can somebody make a release with the updated version of this for Ti SDK 6+?
